airflow/models/dag.py

```diff
@@ -549,6 +550,9 @@ def pickle_id(self):
     def pickle_id(self, value):
         self._pickle_id = value
 
+    def param(self, name, default):
+        return DagParam(self, name, default)
```

Comment:
```suggestion
        return DagParam(dag=self, name=name, default=default)
```
But no strong opinion here